ReflectometryServer/ChannelAccess/pv_manager.py

```diff
@@ -415,7 +415,8 @@ def _add_constants_pvs(self):
                     fields = STANDARD_FLOAT_PV_FIELDS
 
                 self._add_pv_with_fields(prepended_alias, None, fields, beamline_constant.description, None,
-                                         archive=True, interest="MEDIUM", value=value)
+                                         interest="MEDIUM", value=value)
```

Comment:
Are all these PV waveforms, or did we only want `archive=False` for `STANDARD_2048_CHAR_WF_FIELDS` type PVs?

Reply:
No they are not all waveforms, but they are constants so they do not change at runtime (and rarely ever). I write the value to the IOC log once on startup, I didn't think we really lose anything by not archiving them